msprime/_msprimemodule.c

```diff
@@ -1242,6 +1242,29 @@ Simulator_parse_simulation_model(Simulator *self, PyObject *py_model)
     return ret;
 }
 
+static int
+Simulator_parse_activate_population_event(Simulator *self, double time, PyObject *py_event)
+{
+    int ret = -1;
```

Comment:
Need to get some tests into the test_lowlevel file that'll exercise these code paths. See how the other events are done.